packages/http-client-csharp/generator/TestProjects/CadlRanch.Tests/Http/Client/Naming/NamingTests.cs

```diff
@@ -1,109 +1,109 @@
+    //     [CadlRanchTest]
+    //     public Task CompatibleWithEncodedName() => Test(async (host) =>
+    //     {
+    //         var response = await new NamingClient(host, null).CompatibleWithEncodedNameAsync(new ClientNameAndJsonEncodedNameModel(true));
+    //         Assert.AreEqual(204, response.GetRawResponse().Status);
 
-            Assert.NotNull(typeof(ClientNameModel).GetProperty("ClientName"));
-            Assert.IsNull(typeof(ClientNameModel).GetProperty("DefaultName"));
-        });
+    //         Assert.NotNull(typeof(ClientNameModel).GetProperty("ClientName"));
+    //         Assert.IsNull(typeof(ClientNameModel).GetProperty("DefaultName"));
+    //     });
 
-        [CadlRanchTest]
-        public Task Operation() => Test(async (host) =>
-        {
-            var response = await new NamingClient(host, null).ClientNameAsync();
-            Assert.AreEqual(204, response.GetRawResponse().Status);
-        });
+    //     [CadlRanchTest]
+    //     public Task Operation() => Test(async (host) =>
+    //     {
+    //         var response = await new NamingClient(host, null).ClientNameAsync();
+    //         Assert.AreEqual(204, response.GetRawResponse().Status);
+    //     });
 
-        [CadlRanchTest]
-        public Task Parameter() => Test(async (host) =>
-        {
-            var response = await new NamingClient(host, null).ParameterAsync(clientName: "true");
-            Assert.AreEqual(204, response.GetRawResponse().Status);
-        });
+    //     [CadlRanchTest]
+    //     public Task Parameter() => Test(async (host) =>
+    //     {
+    //         var response = await new NamingClient(host, null).ParameterAsync(clientName: "true");
+    //         Assert.AreEqual(204, response.GetRawResponse().Status);
+    //     });
 
-        [CadlRanchTest]
-        public Task Request() => Test(async (host) =>
-        {
-            var response = await new NamingClient(host, null).RequestAsync(clientName: "true");
-            Assert.AreEqual(204, response.GetRawResponse().Status);
-        });
+    //     [CadlRanchTest]
+    //     public Task Request() => Test(async (host) =>
+    //     {
+    //         var response = await new NamingClient(host, null).RequestAsync(clientName: "true");
+    //         Assert.AreEqual(204, response.GetRawResponse().Status);
+    //     });
 
-        [CadlRanchTest]
-        public Task Response() => Test(async (host) =>
-        {
-            var response = await new NamingClient(host, null).ResponseAsync();
-            Assert.IsTrue(response.GetRawResponse().Headers.TryGetValue("default-name", out _));
-            foreach (var header in response.GetRawResponse().Headers)
-            {
-                var key = header.Key;
-                if (key == "default-name")
-                {
-                    var value = header.Value;
-                    Assert.AreEqual("true", value);
-                }
-            }
-        });
+    //     [CadlRanchTest]
```

Comment:
Why are these commented out?

Reply:
because its corresponding source code project has an issue to compile: https://github.com/microsoft/typespec/issues/5566

Reply:
somehow i cannot reproduce the issue :( or it initially does not have it. I will close it.